docs/architecture/grpc-for-wcf-developers/channel-credentials.md

```diff
@@ -117,11 +117,11 @@ class Program
 }
 ```
 
-## Combining ChannelCredentials and CallCredentials
+## Combine channel and call credentials
```

Comment:
I think it's better to keep them as `ChannelCredentials` and `CallCredentials` as they're class names.

```suggestion
## Combine ChannelCredentials and CallCredentials
```